nullaway/src/main/java/com/uber/nullaway/handlers/contract/ContractHandler.java

```diff
@@ -107,110 +171,194 @@ public NullnessHint onDataflowVisitMethodInvocation(
     Preconditions.checkNotNull(analysis);
     Symbol.MethodSymbol callee = ASTHelpers.getSymbol(node.getTree());
     Preconditions.checkNotNull(callee);
-    // Check to see if this method has an @Contract annotation
-    String contractString = ContractUtils.getContractString(callee, config);
-    if (contractString != null && contractString.trim().length() > 0) {
-      // Found a contract, lets parse it.
-      String[] clauses = contractString.split(";");
```

Comment:
The diff is unfortunate here, I hoisted this piece to `ContractUtils` to reuse between methods, which impacts indention/formatting.